instrumentation/jax-rs-1.0/build.gradle

```diff
@@ -14,12 +14,6 @@ jar {
     'Implementation-Title-Alias': 'rest_annotations'}
 }
 
-test {
```

Comment:
Do we know why these were only being used for 9 & 10?   I guess it doesn't really matter since there are no failing tests on 11 or 17. Just seemed a bit odd.

Reply:
I'm not entirely sure but I was wondering the same thing. I verified that the tests pass on later versions of Java but I don't know what was causing issues with 9/10 specifically. I also saw a number of tests that had `JavaNNIncompatibleTest` annotations on them up to Java 16/17 but then no longer had them for Java 18/19 but they mysteriously worked on these new Java versions, which I wouldn't have expected.